packages/langium/src/references/scope-computation.ts

```diff
@@ -75,11 +79,26 @@ export class DefaultScopeComputation implements ScopeComputation {
     }
 
     async computeExports(document: LangiumDocument, cancelToken = CancellationToken.None): Promise<AstNodeDescription[]> {
+        return this.computeExportsForNode(document.parseResult.value, document, undefined, cancelToken);
+    }
+
+    /**
+     * Creates {@link AstNodeDescription AstNodeDescriptions} for the given @link AstNode parentNode} and its children.
```

Comment:
These `@link` tags don't seem to work in VS Code. I'd propose to use `` `code-style` `` instead.

Reply:
For me they worked actually, otherwise I wouldn't have used them ... I have a quite brand new version

Reply:
![image](https://user-images.githubusercontent.com/8513883/190630345-7722e6d5-20d5-4fff-b0b8-eac090b16348.png)

Ok, they click experience is not ultimately perfect yet...

Reply:
Just missed a `{` ... damn.